src/Reset.ts

```diff
@@ -870,13 +870,6 @@ export const singularity = async (): Promise<void> => {
     const hold = Object.assign({}, blankSave, {
         codes: Array.from(blankSave.codes)
     }) as Player;
-    //Reset Displays
-    toggleTabs('buildings');
-    toggleSubTab(1, 0);
-    toggleSubTab(4, 0); // Set 'runes' subtab back to 'runes' tab
-    toggleSubTab(8, 0); // Set 'cube tribues' subtab back to 'cubes' tab
-    toggleSubTab(9, 0); // set 'corruption main'
-    toggleSubTab(-1, 0); // set 'statistics main'
```

Comment:
If these don't reset you can end up having access to tabs which are otherwise not unlocked until later in the singularity. Keep these displays please

Reply:
I returned it as you pointed out